libmamba/src/core/util.cpp

```diff
@@ -246,6 +246,24 @@ namespace mamba
         return strip(input, WHITESPACES);
     }
 
+    // todo one can deduplicate this code with the one in strip() by using C++ 20 with concepts
+    std::wstring_view wstrip(const std::wstring_view& input)
+    {
+        return wstrip(input, WHITESPACES_WSTR);
+    }
+
+    std::wstring_view wstrip(const std::wstring_view& input, const std::wstring_view& chars)
```

Comment:
You didn't find a way to use templating for `strip`?

Reply:
You are right, the only thing that doesn't work is templating when using `WHITESPACES` or `WHITESPACES_WSTR` because they have a fixed type... But `strip can be deduplicated`. 👍🏻